libs/base/ui/navigation/navigation-item/src/navigation-item.styles.ts

```diff
@@ -42,6 +49,24 @@ export const navigationItemStyles = css`
     border-radius: 4px;
   }
 
+  :host([divider])::after {
+    content: '';
+    background-color: var(--oryx-color-neutral-5);
+    position: absolute;
+    width: 80%;
+    height: 1px;
+    transform: translateY(var(--oryx-navigation-item-divider-position));
+  }
+
+  :host([divider]) {
+    margin-block-start: calc(var(--oryx-navigation-item-vertical-padding) * 2);
+  }
+
+  :host([divider]:host-context([collapsed]))::after {
+    transform: translateY(var(--oryx-navigation-item-divider-position))
+      translateX(-33%);
+  }
+
```

Comment:
Does this work on Firefox? It's displaying with the wrong horizontal margins in at least storybook there.